### PR TITLE
Clarify headerBackTitle usage

### PR DESCRIPTION
**Pull request by njt1982**
Clarifying something in the documentation which I only learned by searching the closed issue queue :)

https://github.com/react-community/react-navigation/issues/1358#issuecomment-298918466

### CODE REVIEW

File: docs/api/navigators/StackNavigator.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ String or React Element used by the header. Defaults to scene `title`
 
 #### `headerBackTitle`
 
-Title string used by the back button on iOS or `null` to disable label. Defaults to scene `title`
+Title string used by the back button on iOS or `null` to disable label. Defaults to scene `title`. The back button will use the `headerBackTitle` value from the previous screen, not the current screen.
```
Comment (pvinis) [inline docs/api/navigators/StackNavigator.md]: maybe this is better? `Title string used by the back button on iOS, or `null` to disable label. Defaults to the previous scene's `headerTitle`.

Comment (skevy) [inline docs/api/navigators/StackNavigator.md]: Yah @njt1982 could you change it to @pvinis's suggestion? I think it reads better.

Thanks!

Comment (njt1982) [inline docs/api/navigators/StackNavigator.md]: @pvinis @skevy done in 44fbb3f

**Comment by skevy** [comment]
Thanks!

**Pull request merged**